### PR TITLE
{bio} [goolf-1.4.10] SCANMS-2.02  (REVIEW)  

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**SUCCESS**
Build succeeded for 1 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/454a6c6a0d38d768e8b4 for a full test report.

File: easybuild/easyconfigs/s/SCANMS/SCANMS-2.02-goolf-1.4.10-Perl-5.16.3.eb [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+]
+
+# workaround so sanity check in the PerlModule easyblock works. 
+# Just check the existence of a perl core module
+options = {'modulename': 'POSIX'}
+
+sanity_check_paths = {
+        'files': ["bin/scanms"],
+        'dirs': [""]
+}
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/s/SCANMS/SCANMS-2.02-goolf-1.4.10-Perl-5.16.3.eb]: Maybe shuffle some white space, it looks good:
```
    'files': ["bin/scanms"],
    'dirs': []
}```

File: easybuild/easyconfigs/s/SCANMS/SCANMS-2.02-goolf-1.4.10-Perl-5.16.3.eb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+# copy all the documentation provided in the tarball to install directory
+postinstallcmds = ["cp -r %(builddir)s/*/{README,SCANMS.pdf,scanms.doc.html} %(installdir)s"]
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/s/SCANMS/SCANMS-2.02-goolf-1.4.10-Perl-5.16.3.eb]: nitpicking: space after comma

Comment (boegel) [inline easybuild/easyconfigs/s/SCANMS/SCANMS-2.02-goolf-1.4.10-Perl-5.16.3.eb]: hmm, that may not work

File: easybuild/easyconfigs/s/SCANMS/SCANMS-2.02-goolf-1.4.10-Perl-5.16.3.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/s/SCANMS/SCANMS-2.02-goolf-1.4.10-Perl-5.16.3.eb]: +1 on this

File: easybuild/easyconfigs/s/SCANMS/SCANMS-2.02-goolf-1.4.10-Perl-5.16.3.eb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+perlver = '5.16.3'
+versionsuffix = '-%s-%s' % (perl, perlver)
+
+dependencies = [
+    (perl, perlver),
+]
+
+# workaround so sanity check in the PerlModule easyblock works. 
+# Just check the existence of a perl core module
+options = {'modulename': 'POSIX'}
```
Comment (boegel) [inline easybuild/easyconfigs/s/SCANMS/SCANMS-2.02-goolf-1.4.10-Perl-5.16.3.eb]: wait, what's this about?

You should redefine `exts_filter` instead, see https://github.com/hpcugent/easybuild-easyconfigs/search?utf8=%E2%9C%93&q=exts_filter

File: easybuild/easyconfigs/s/SCANMS/SCANMS-2.02-goolf-1.4.10-Perl-5.16.3.eb [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+sanity_check_paths = {
+        'files': ["bin/scanms"],
+        'dirs': []
+}
+
+# copy all the documentation provided in the tarball to install directory
+postinstallcmds = ["cp -r %(builddir)s/*/{README,SCANMS.pdf,scanms.doc.html} %(installdir)s"]
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/s/SCANMS/SCANMS-2.02-goolf-1.4.10-Perl-5.16.3.eb]: space after comma, if it doesn't break the functionality?

Comment (pescobar) [inline easybuild/easyconfigs/s/SCANMS/SCANMS-2.02-goolf-1.4.10-Perl-5.16.3.eb]: @fgeorgatos I tried it and it crashes

Comment (fgeorgatos) [inline easybuild/easyconfigs/s/SCANMS/SCANMS-2.02-goolf-1.4.10-Perl-5.16.3.eb]: I can see why, it has to do with the curly brackets operators under shell, thanks for trying it though!

File: easybuild/easyconfigs/s/SCANMS/SCANMS-2.02-goolf-1.4.10-Perl-5.16.3.eb [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+dependencies = [
+    (perl, perlver),
+]
+
+# override the default sanity check for perl packages which tries to load Perl module
+exts_filter = ('scanms -v 2>&1 | grep version', "")
+
+sanity_check_paths = {
+        'files': ["bin/scanms"],
+        'dirs': []
```
Comment (boegel) [inline easybuild/easyconfigs/s/SCANMS/SCANMS-2.02-goolf-1.4.10-Perl-5.16.3.eb]: funky indent

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/4bcb1c028be50b6ebcae for a full test report.

**Comment by boegel** [comment]
Going in, thanks @pescobar!

**Pull request merged**